mimic/rest/maas_api.py

```diff
@@ -2,7 +2,7 @@
 MAAS Mock API
 """
 
-from __future__ import division, unicode_literals
+from __future__ import division, unicode_literals, print_function
```

Comment:
+1 for removing noise. It makes no difference to me whether you use print statements for debugging or not, but we shouldn't have print statements in our production code and I see `print_function` and immediately think *someone used a print in this module!*

Logging is better than printing.